.travis.yml

```diff
@@ -33,31 +34,41 @@ matrix:
     - php: 7
       env:
         - DEPS=latest
-    - php: hhvm 
+    - php: 7.1
       env:
         - DEPS=lowest
-    - php: hhvm 
+    - php: 7.1
       env:
         - DEPS=locked
-    - php: hhvm 
+    - php: 7.1
+      env:
+        - DEPS=latest
+    - php: 7.2
+      env:
+        - DEPS=lowest
+    - php: 7.2
+      env:
+        - DEPS=locked
+    - php: 7.2
       env:
         - DEPS=latest
   allow_failures:
-    - php: hhvm
+    - php: 7.2
   
 notifications:
   irc: "irc.freenode.org#apigility-dev"
   email: false
 
 before_install:
-  - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
+  - if [[ $TEST_COVERAGE != 'true' && ! $TRAVIS_PHP_VERSION =~ ^7.2 ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
```

Comment:
Is it a big issue? I can see only:
```
$ if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
File xdebug.ini does no exist.
```
in travis logs, and build is not failing because of that.